README.md

```diff
@@ -1,4 +1,196 @@
-Astarte Dashboard
-=================
+# Astarte Dashboard
 
-Astarte Dashboard allows users perform administrative tasks such as installing interfaces and triggers.
+## Table of Contents
```

Comment:
I still don't like the ToC here, since you are forcing users to scroll the page to discover what the dashboard is.
I think that the most important information should be visible at glance, and right now what I see here is just the ToC which doesn't tell me anything about the dashboard.